components/time-picker/locale/fr_FR.tsx

```diff
@@ -2,6 +2,7 @@ import { TimePickerLocale } from '../index';
 
 const locale: TimePickerLocale = {
   placeholder: "Sélectionner l'heure",
+  rangePlaceholder: ['Heure de début', 'Heure de fin'],
```

Comment:
How about move this line to `date-picker/locale` ?

Reply:
I preferred to copy the implementation of `components/time-picker/locale/en_US.tsx` and the placeholder is related to the _time-picker_